packages/bot/src/helpers.ts

```diff
@@ -976,7 +976,7 @@ export type BotHelpers<TProps extends TransformersDesiredProperties, TBehavior e
   getChannels: (guildId: BigString) => Promise<SetupDesiredProps<Channel, TProps, TBehavior>[]>
   getChannelWebhooks: (channelId: BigString) => Promise<SetupDesiredProps<Webhook, TProps, TBehavior>[]>
   getDmChannel: (userId: BigString) => Promise<SetupDesiredProps<Channel, TProps, TBehavior>>
-  getGroupDmChannel: (options: GetGroupDmOptions) => Promise<SetupDesiredProps<Channel, TProps, TBehavior>>
+  getGroupDmChannel: (options: CreateGroupDmOptions) => Promise<SetupDesiredProps<Channel, TProps, TBehavior>>
```

Comment:
Should we rename these to create DM channel and create group DM channel to make it consistent with the docs? It's kinda jarring that the function is called get, but the options interface (and discord docs) are create

Reply:
Opened #4383